src/number_theory/pell.lean

```diff
@@ -38,6 +40,34 @@ Pell's equation
 * Connect solutions to the continued fraction expansion of `√d`.
 -/
 
+namespace zsqrtd
+
+-- Note: We put these here to avoid having to import `algebra.star.unitary`
+--       into `number_theory.zsqrtd.basic` just for these two results.
```

Comment:
I don't know. If it is OK to add this import to `number_theory.zsqrtd.basic`, the results can be moved there.

Reply:
I think it's fine to move. It's not that heavy of an import.

Reply:
OK. I think I will leave `is_pell_solution_iff_mem_unitary` in the `pell` file, though (and put it in the `pell` namespace, since it is more specific to Pell's equation.